tests/compile-and-dump/Singletons/FunctorLikeDeriving.ghc88.template

```diff
@@ -169,16 +169,16 @@ Singletons/FunctorLikeDeriving.hs:(0,0)-(0,0): Splicing declarations
                                                               arg. SameKind (Apply Lambda_0123456789876543210Sym0 arg) (Lambda_0123456789876543210Sym1 arg) =>
                                                        Lambda_0123456789876543210Sym0 _f_01234567898765432100123456789876543210
     type instance Apply Lambda_0123456789876543210Sym0 _f_01234567898765432100123456789876543210 = Lambda_0123456789876543210Sym1 _f_01234567898765432100123456789876543210
-    type family Fmap_0123456789876543210 (a :: (~>) a b) (a :: T x a) :: T x b where
+    type family Fmap_0123456789876543210 (a :: (~>) a0 b0) (a :: T x a0) :: T x b0 where
       Fmap_0123456789876543210 _f_0123456789876543210 (MkT1 a_0123456789876543210 a_0123456789876543210 a_0123456789876543210 a_0123456789876543210) = Apply (Apply (Apply (Apply MkT1Sym0 (Apply (Apply (Apply (Apply (Apply (Apply Lambda_0123456789876543210Sym0 _f_0123456789876543210) a_0123456789876543210) a_0123456789876543210) a_0123456789876543210) a_0123456789876543210) a_0123456789876543210)) (Apply _f_0123456789876543210 a_0123456789876543210)) (Apply (Apply FmapSym0 _f_0123456789876543210) a_0123456789876543210)) (Apply (Apply FmapSym0 (Apply FmapSym0 _f_0123456789876543210)) a_0123456789876543210)
       Fmap_0123456789876543210 _f_0123456789876543210 (MkT2 a_0123456789876543210) = Apply MkT2Sym0 (Apply (Apply (Apply Lambda_0123456789876543210Sym0 _f_0123456789876543210) a_0123456789876543210) a_0123456789876543210)
-    type Fmap_0123456789876543210Sym2 (a0123456789876543210 :: (~>) a0123456789876543210 b0123456789876543210) (a0123456789876543210 :: T x0123456789876543210 a0123456789876543210) =
+    type Fmap_0123456789876543210Sym2 (a0123456789876543210 :: (~>) a01234567898765432109 b01234567898765432109) (a0123456789876543210 :: T x0123456789876543210 a01234567898765432109) =
```

Comment:
Again, this isn't directly your fault, but it's rather curious that these unique numbers became larger all of a sudden. I wonder if there's some underlying explanation for why this is happening? (If you don't have an answer for this question, don't worry about it too much.)

Reply:
This happened just from using GHC 8.8, but I haven't bisected what GHC change caused it.

Reply:
One possibility is that something related to GHC's generation of unique names changed between 8.6 and 8.8. A similar increase in unique number size happened between 8.0 and 8.2 when [this commit] landed (see [here](https://github.com/goldfirere/singletons/commit/637fb4d0435220f740bc658973f7a05a499dcee6#diff-7b178c8c1e207617509cdfd8802845faR4) for an example).